app/javascript/elm/src/Main.elm

```diff
@@ -351,29 +378,7 @@ view model =
                                 )
                             ]
                             [ div [ Attr.class "sessions", Attr.attribute "ng-controller" "SessionsGraphCtrl" ]
-                                [ div [ Attr.attribute "ng-controller" "SessionsListCtrl" ]
-                                    (case model.selectedSessionId of
-                                        Nothing ->
-                                            [ h2 [ Attr.class "sessions-header" ]
-                                                [ text "Sessions" ]
-                                            , span [ Attr.class "sessions-number" ]
-                                                [ text "showing 6 of 500 reuslts" ]
-                                            , viewSessions model
-                                            ]
-
-                                        Just _ ->
-                                            [ div [ Attr.class "single-session-container" ]
-                                                [ div [ Attr.class "single-session-info" ]
-                                                    [ p [ Attr.class "single-session-owner" ] [ text "NYCEJA" ] ]
-                                                , div
-                                                    [ Attr.class "single-session-graph", Attr.id "graph-box" ]
-                                                    [ div [ Attr.id "graph" ] []
-                                                    ]
-                                                , div [ Attr.class "single-session-close" ] [ button [ Events.onClick DeselectSession ] [ text "X" ] ]
-                                                ]
-                                            ]
-                                    )
-                                ]
+                                [ div [ Attr.attribute "ng-controller" "SessionsListCtrl" ] (viewSessionsOrSelectedSession model.selectedSession model.sessions) ]
```

Comment:
how cute :)

elm and angular sitting in a tree
k-i-s-s-i-n-g

Reply:
We should add React and make it a threesome ;)